syntax = "proto3";

service ExampleService {
  rpc classify(ClassifyRequest) returns (ClassifyResponse);
}

message ClassifyRequest {
  string image_type = 1;
  string image = 2;
}

message ClassifyResponse {
  repeated string predictions = 1;  
  repeated float confidences = 2;
  bytes binary_field = 3;
}

